js_modules/dagit/packages/core/src/ui/Filters/TimeRangeFilter.tsx

```diff
@@ -0,0 +1,291 @@
+import {IconName, Box, Icon, Colors, Dialog, Button, DialogFooter} from '@dagster-io/ui';
+import dayjs from 'dayjs';
+import isEqual from 'lodash/isEqual';
+// eslint-disable-next-line no-restricted-imports
+import moment from 'moment';
+import React from 'react';
+import {DateRangePicker} from 'react-dates';
+import styled from 'styled-components/macro';
+
+import {Filter, FilterTag, FilterTagHighlightedText} from './Filter';
+
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+
+function calculateTimeRanges() {
+  return {
+    TODAY: {
+      label: 'Today',
+      range: [dayjs().startOf('day').toDate().getTime(), null] as TimeRangeState,
```

Comment:
See earlier comment re: moment-tz. If a user is using a different timezone in our app than the default browser timezone, these ranges might be incorrect.